src/binary/coreir.cpp

```diff
@@ -267,5 +269,8 @@ int main(int argc, char* argv[]) {
   }
   LOG(DEBUG) << "Modified?: " << (modified ? "Yes" : "No");
 
+  if (delete_sout) delete sout;
+  delete c;
```

Comment:
Prefer: `deleteContext(c);`

(Not a huge fan of this pattern anyway, but since it's used elsewhere might as well keep it standard)